google/ggrc-core
# GGRC-5539 Internal (not defined in API) fields are sent with POST and PUT requests with Assessment Templates

# Issue description

Remove extra fields which the application sends for Assessment Template based on investigation

# Steps to test the changes

1. Create Assessment Template.
2. Change Assessment Template.

**Actual result**: extra fields sends for POST and PUT requests.

Extra fields for POST requests: `assigneesList, assigneesListDisable, people_values, showCaptainAlert, verifiersList, verifiersListDisable`.

Extra fields for PUT requests: `assigneesList, verifiersList, people_values, showCaptainAlert, assigneesListDisable, verifiersListDisable`

**Expected result**:  extra fields don't sends for POST and PUT requests.

# Solution description

Create `people-list` components and move some fields and logic from  `AssessmentTemplate` model.

# Sanity checklist




# PR Review checklist
